spec/components/app_session_needs_review_warning_component_spec.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+describe AppSessionNeedsReviewWarningComponent do
+  subject(:rendered) { render_inline(component) }
```

Comment:
It feels like this subject belongs in the `describe "rendered content"` block as that's the subject of the tests in there.